app/controllers/recipients_controller.rb

```diff
@@ -0,0 +1,3 @@
+class RecipientsController < ApplicationController
+    def index; end
```

Comment:
the indentation in this file is inconsistent with the others. We use 2 spaces in Ruby.